polaris-core/src/test/java/org/apache/polaris/service/storage/StorageLocationTest.java

```diff
@@ -19,17 +19,41 @@
 package org.apache.polaris.service.storage;
 
 import org.apache.polaris.core.storage.StorageLocation;
-import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 
 public class StorageLocationTest {
 
   @Test
-  public void testOfDifferentPrefixes() {
-    StorageLocation StandardLocation = StorageLocation.of("file:///path/to/file");
+  public void testAwsLocations() {
+    StorageLocation baseStorageLocation = StorageLocation.of("s3://path/to/file");
+    StorageLocation testStorageLocation = StorageLocation.of("s3://path/to/////file");
+    Assertions.assertEquals(testStorageLocation, baseStorageLocation);
```

Comment:
This test is good, but I believe more test cases are needed to validate the handling of upper / lower case letters and leading / trailing slashes and the special cases of missing scheme.

Reply:
Good call out - let me add them